### ISSUE TITLE
Setting --cutoff to zero crashes RnaChipIntegrator

### ISSUE THREAD

**Issue by pjbriggs** [fls-bioinformatics-core/RnaChipIntegrator#41]
Using `--cutoff=0` should disable the distance cutoff limit but instead it causes the program to crash:

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.6.9/bin/RnaChipIntegrator", line 9, in <module>
    load_entry_point('RnaChipIntegrator==1.0.0', 'console_scripts', 'RnaChipIntegrator')()
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/rnachipintegrator/cli.py", line 235, in main
    print "Maximum cutoff distance: %d (bp)" % max_distance
TypeError: %d format: a number is required, not NoneType
```

**Comment by pjbriggs** [comment]
Fixed by PR #43.

**Comment by pjbriggs** [comment]
This problem is not fixed after all - it still manifests if `--cutoff=0` is used in conjunction with `--xlsx`, in which case the crash happens when trying to write to the XLSX spreadsheet:

    Traceback (most recent call last):
      File "/home/pjb/scratchpad/test_rnachipintegrator/rcivenv/bin/RnaChipIntegrator", line 9, in <module>
        load_entry_point('RnaChipIntegrator==1.0.1', 'console_scripts', 'RnaChipIntegrator')()
      File "/home/pjb/scratchpad/test_rnachipintegrator/rcivenv/local/lib/python2.7/site-packages/rnachipintegrator/cli.py", line 373, in main
        xlsx.append_to_notes("Maximum cutoff distance (bp)\t%d" % max_distance)
    TypeError: %d format: a number is required, not NoneType
